src/averageAges.js

```diff
@@ -14,12 +16,28 @@
  *
  * @return {number}
  */
+
+//   // write code here
+//   // learn how to use array methods like
+//  .filter .map .some .every .find .reduce
+//   // avoid using loop and forEach
+//   // replace `if ()` statement with &&, || or ?:
+//   // without nesting
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const MEN = people.filter((person) => person.sex === 'm');
+  const MEN_IN_CENTURY = MEN
+    .filter((person) => Math.ceil(person.died / 100) === century);
+  const ALL_MEN_IN_CENTURY_AGE = MEN_IN_CENTURY
+    .map(person => (person.died - person.born));
+  const AGES_SUM = ALL_MEN_IN_CENTURY_AGE.reduce((a, b) => a + b, 0);
+  const AVERAGE_AGE_WITH_CENTURY = AGES_SUM / ALL_MEN_IN_CENTURY_AGE.length;
+  const AVERAGE_MEN_AGE = MEN.map(person => (person.died - person.born))
```

Comment:
typically we use uppercased variables for hardcoded values, these are calculated